chartpress.yaml

```diff
@@ -0,0 +1,14 @@
+charts:
```

Comment:
There is already a chartpress file in helm-chart/amalthea

Reply:
I dont know what the deploy action needs. 

Either way we should definitely not make copies of this file. Either move the file to where the actions are happy or symlink.